src/seedu/addressbook/data/person/Grades.java

```diff
@@ -2,67 +2,62 @@
 
 import java.util.Objects;
 
-import seedu.addressbook.data.exception.IllegalValueException;
+//import seedu.addressbook.data.exception.IllegalValueException;
+
 /**
- * Represents student's grade.s
+ * Represents student's values
  */
 public class Grades implements Printable {
 
     public static final String GRADE_EXAMPLE = "27";
 
-    public static final String GRADE_CONSTRAINTS = "May include up to decimal places only";
-    public static final String GRADE_VALIDATION_REGEX = "\\d+(\\.\\d+)?";
+    //public static final String MESSAGE_GRADE_CONSTRAINTS = "May include up two decimal places only";
 
-    private String grade;
-    //private Person person;
+    private int value;
 
-    /**
-     * Validates given results.
-     *
-     * @throws IllegalValueException if given results string is invalid.
-     */
-    public Grades(String grade) throws IllegalValueException {
-
-        String trimmedMarksScored = grade.trim();
-        if (!isValidMarksScored(trimmedMarksScored)) {
-            throw new IllegalValueException(GRADE_CONSTRAINTS);
-        }
-        // Assign grade with score
-        this.grade = trimmedMarksScored;
-    }
+    private Person person;
 
```

Comment:
Hmm since you are using a map now, is there a need for Grade object to keep track of who it belongs to?